terraform/tap.tf

```diff
@@ -50,32 +50,30 @@ resource "pacific_guestcluster" "gc" {
   version                            = "v1.21.2"
```

Comment:
The other changes look good, but I note that 1.21 is no longer supported for TAP 1.2